cfme/utils/template/template_upload.py

```diff
@@ -76,7 +76,7 @@ def parse_cmd_line():
 
 def get_stream_from_image_url(image_url, quiet=False):
     """Get default image URL for a given stream name"""
-    # strip trailing / from URL, and strip build number or link (5.9.2.3, latest, stable)
+    # strip trailing / from URL, and strip build number or link (5.11.0.1, latest, stable)
```

Comment:
Q: do we want to update this comment each time new build is released?

Reply:
no, and ultimately it doesn't really need to be updated at all since its just a comment.